docs/src/extend/index.md

```diff
@@ -21,34 +21,26 @@ In order to work with ESLint as a developer, it's recommended that:
 
 If that sounds like you, then continue reading to get started.
 
-## Section 1: Get the [Source Code](../contribute/source-code)
+## [Create Plugins](plugins)
 
-Before you can get started, you'll need to get a copy of the ESLint source code. This section explains how to do that and a little about the source code structure.
-
-## Section 2: Set up a [Development Environment](../contribute/development-environment)
-
-Developing for ESLint is a bit different than running it on the command line. This section shows you how to set up a development environment and get you ready to write code.
-
-## Section 3: Run the [Unit Tests](../contribute/tests)
-
-There are a lot of unit tests included with ESLint to make sure that we're keeping on top of code quality. This section explains how to run the unit tests.
+You've developed library-specific rules for ESLint and you want to share them with the community. You can publish an ESLint plugin on npm.
 
-## Section 4: [Working with Rules](custom-rules)
+## [Custom Rules](custom-rules)
 
-You're finally ready to start working with rules. You may want to fix an existing rule or create a new one. This section explains how to do all of that.
+This section explains how to create and modify rules to use with ESLint.
 
-## Section 5: [Working with Plugins](plugins)
+## [Custom Formatters](custom-formatters)
 
-You've developed library-specific rules for ESLint and you want to share them with the community. You can publish an ESLint plugin on npm.
+This section explains how you can create a custom formatter to control what ESLint outputs.
 
-## Section 6: [Working with Custom Parsers](custom-parsers)
+## [Custom Parsers](custom-parsers)
 
 If you aren't going to use the default parser of ESLint, this section explains about using custom parsers.
```

Comment:
While here, I think we should update this description because the linked page explains how to create custom parsers, not how to use custom parsers.

Reply:
addressed in aaad16323